src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -99,7 +99,7 @@
 #include "transformations/rt_info/keep_const_precision.hpp"
 #include "transformations/transpose_sinking/ts_shape_of.hpp"
 #include "utils/ngraph_transformation.hpp"
-#include "utils/print_model.hpp"
```

Comment:
it's better to move it transformations/utils/print_model.hpp
all other transformations are used in some pipelines, this transformation is more for debugging and development purposes

Reply:
Ok, moved to _transformations/utils/_